File: TypeScript/Employee.ts
export class Employee{
    empID:number=123;
    empName:string="Santosh";
    empSal:number=24714;
    constructor(i:number,n:string,s:number){
        this.empID=i;
        this.empName=n;
        this.empSal=s;
    }
    display(){
        console.log(this.empID);
        console.log(this.empName);
        console.log(this.empSal);
    }


}

var e1=new Employee(122,"ramesh",23456);
e1.display();

//var e1=new Employee();
//e1.display();

File: CoreJava/Day3_assignment/src/com/sonata/Employee.java
package com.sonata;

public class Employee {
 int empId;
 String name;
 double salary;
 Employee(int empid,String name,double salary){
	this.empId=empid;
	this.name=name;
	this.salary=salary;
}
 public Employee() {
}
 public double cal(double hra,double da,double ta){
		return this.salary+hra+da+ta;
	}
 public void display() {
	 System.out.println(empId);
	 System.out.println(name);
	 System.out.println(salary);
 }
 public static void main(String args[]) {
	 
	 Employee e=new Employee();
	 e.empId=21018;
	 e.name="santosh";
	 e.salary=27842;
	 e.display();
	 
 }
}
